src/Model/EloquentModel.php

```diff
@@ -79,6 +79,13 @@ class EloquentModel extends Model implements Arrayable, PresentableInterface
      */
     protected $cascades = [];
 
+    /**
+     * The restricting delete-able relations.
+     *
+     * @var array
+     */
+    protected $restrict = [];
```

Comment:
I suppose it should be `$restricted` or `$restricts`. But it is better to ask Ryan about it...

Reply:
I guess `$restricts` is better, we already use `$cascades` for casdades. @RyanThompson your call.